localization/ndt_scan_matcher/README.md

```diff
@@ -228,9 +228,9 @@ Here is a split PCD map for `sample-map-rosbag` from Autoware tutorial: [`sample
 | single file |           true            |            true            |  at once (standard)  |
 | single file |           true            |           false            |  **does NOT work**   |
 | single file |           false           |         true/false         |  at once (standard)  |
-|  splitted   |           true            |            true            |     dynamically      |
-|  splitted   |           true            |           false            |  **does NOT work**   |
-|  splitted   |           false           |         true/false         |  at once (standard)  |
+| split files |           true            |            true            |     dynamically      |
+| split files |           true            |           false            |  **does NOT work**   |
+| split files |           false           |         true/false         |  at once (standard)  |
```

Comment:
The adjective usage of `split` is not listed in some dictionaries, so it feels a little unnatural in English for me.
Also, `splitted` is natural English, so why not consider adding it to the dictionary?

Reply:
@HansRobo 
Thank you for your comment!

> adjective usage

No, it's not adjective, but past participle. Split files is the same form as broken files.
In addition, IMO, _Splitted_ is not standard as past participle of _split_. Please check this page https://english.stackexchange.com/questions/467873/the-past-participle-of-split-split-or-splitted

Reply:
Oh, I see!
English is difficult...

Reply:
@Shin-kyoto @HansRobo By the way, Does the word `multiple files` not work here? I think it's easier to understand because another word is `single file`.

Reply:
@kenji-miyake 
I chose your plan in https://github.com/autowarefoundation/autoware.universe/pull/3416/commits/5c085ea0bd796707be2ccb90c74c0b8b7f5b90b7, thank you for your advice!